MoneyWise2020/settings.py

```diff
@@ -67,16 +69,20 @@
     },
 ]
 
-WSGI_APPLICATION = 'src.wsgi.application'
+WSGI_APPLICATION = 'MoneyWise2020.wsgi.application'
 
 
 # Database
 # https://docs.djangoproject.com/en/3.1/ref/settings/#databases
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'MoneyWise',
+        'USER': 'admin',
+        'PASSWORD': 'agile2020',
+        'HOST': 'moneywise-database-1.c1idjdax3vgr.us-east-1.rds.amazonaws.com',
+        'PORT': '3306',        
```

Comment:
Best if these are environment variables too. Would like to be able to run mysql locally so we can work independently and quickly. Particularly important when things like migrations get involved.